plugins/polio/js/config.js

```diff
@@ -588,7 +588,9 @@ const menu = [
                         label: MESSAGES.vaccineSupplyChain,
                         key: 'supplychain',
                         dev: true,
```

Comment:
Is this menu will stay in dev mode?

Reply:
No it shouldn't, thanks fro pointing it out